policy/approval/approve.go

```diff
@@ -430,19 +450,32 @@ func isIgnoredCommit(ctx context.Context, prctx pull.Context, actors *common.Act
 	return len(c.Users()) > 0, nil
 }
 
-func findLastPushed(commits []*pull.Commit) *pull.Commit {
-	var last *pull.Commit
-	for _, c := range commits {
-		if c.PushedAt != nil && (last == nil || c.PushedAt.After(*last.PushedAt)) {
-			last = c
-		}
-	}
-	return last
-}
-
 func numberOfApprovals(count int) string {
 	if count == 1 {
 		return "1 approval"
 	}
 	return fmt.Sprintf("%d approvals", count)
 }
+
+// sortCommits orders commits in history order starting from head. It must be
+// called on the unfiltered set of commits.
+func sortCommits(commits []*pull.Commit, head string) []*pull.Commit {
```

Comment:
I _think_ this is unnecessary because GitHub already returns the commits in this order, but I'd rather be explicit than have this break strangely if the order changes for some reason - this ordering is important for push detection to work in all cases.